user_docs/en/userGuide.t2t

```diff
@@ -592,12 +592,12 @@ A key command is provided to return to the original page containing the embedded
 + Reading Mathematical Content +[ReadingMath]
 Using MathPlayer 4 from Design Science, NVDA can read and interactively navigate supported mathematical content.
 This requires that MathPlayer 4 is installed on the computer.
-MathPlayer is available as a free download from: http://www.dessci.com/en/products/mathplayer/
+MathPlayer is available as a free download from: https://www.dessci.com/en/products/mathplayer/
 
 NVDA supports the following types of mathematical content:
 - MathML in Mozilla Firefox, Microsoft Internet Explorer and Google Chrome.
 - Design Science MathType in Microsoft Word and PowerPoint.
-MathType needs to be installed in order for this to work. The trial version is sufficient.
+MathType needs to be installed in order for this to work. The trial version is sufficient. It can be downloaded from https://www.dessci.com/en/products/mathtype/

```

Comment:
Can you put this new sentence on a new line please. Single line breaks are removed when this is rendered. We separate sentences to minimize translations. If you put this new sentence on a new line, then it will only be the new sentence and not three that require re-translation.